tests/test_decoder.py

```diff
@@ -142,6 +143,7 @@ def test_decoder_for_rule(self):
         self.assertEquals(decoded.alert_by, 'any')
         self.assertEquals(decoded.key, 'foo')
         self.assertEquals(len(decoded.conditions), 1)
+        self.assertEquals(decoded.status, 'logonly')
```

Comment:
The point of defining the constant is to hide that detail from the user.  If we are going to expose the detail anyway then we can delete the constants